src/components/navigator.js

```diff
@@ -764,7 +784,7 @@ export default class Navigator {
     if (evt.altKey || evt.ctrlKey || evt.metaKey || evt.shiftKey)
       return
 
-    if (!this._pause && this._sectionCount && getKeyMapping(evt.keyCode) === 'center') {
+    if (!this._pause && this._sectionCount && this._mapToKeyName(evt.keyCode) === KeyName.ENTER) {
```

Comment:
There was a bug here. `center` was never a valid key name. It's a type for `enter`.